appshell/cef_window.h

```diff
@@ -74,6 +74,9 @@ static __inline int RectHeight(const RECT &rIn)
 #define DCX_USESTYLE 0x00010000
 #endif
 
+//HiDPI The default logical DPI when scaling is applied in windows. see. https://msdn.microsoft.com/en-us/library/ms701681(v=vs.85).aspx
+#define DEFAULT_WINDOWS_DPI 96  
```

Comment:
Can we move this to the CPP counter part. That way this variable would very localized.